lib/grandpa/message_handler.go

```diff
@@ -91,7 +92,24 @@ func (h *MessageHandler) handleNeighbourMessage(msg *NeighbourMessage) error {
 }
 
 func (h *MessageHandler) handleCommitMessage(msg *CommitMessage) error {
-	logger.Debugf("received commit message %v", msg)
+	logger.Debugf("received commit message, msg: %+v", msg)
+
+	containsPrecommitsSignedBy := make([]string, len(msg.AuthData))
+	for i, authData := range msg.AuthData {
+		containsPrecommitsSignedBy[i] = authData.AuthorityID.String()
+	}
+
+	err := telemetry.GetInstance().SendMessage(
+		telemetry.NewAfgReceivedCommitTM(
+			msg.Vote.Hash,
+			fmt.Sprintf("%d", msg.Vote.Number),
+			containsPrecommitsSignedBy,
+		),
+	)
+	if err != nil {
+		logger.Debugf("problem sending afg.received_commit telemetry message, err: %s", err)
```

Comment:
nit

```suggestion
		logger.Debugf("problem sending afg.received_commit telemetry message: %s", err)
```

Should this be a warning instead? Or does this happen often?